workflows/scripts/ascat/run_ascat.R

```diff
@@ -0,0 +1,43 @@
+library(ASCAT)
+library(data.table)
+
+# Get command line arguments
+args <- commandArgs(trailingOnly = TRUE)
+
+# Define variables
+tumorid = args[1]
+tumorLogR = args[2]
+tumorBAF = args[3]
+normalLogR = args[4]
+normalBAF = args[5]
+GCcontentfile = args[6]
+replictimingfile = args[7]
+output = dirname(args[8])
+
+gender = "XX"
```

Comment:
Does it always use gender = XX? it doesn't change with the sample?

Reply:
I had no luck trying with XY, I also could not find the reference files in ASCAT for chrY, but that is maybe beside the point: https://github.com/VanLoo-lab/ascat/tree/master/ReferenceFiles/WGS